tendermint/src/rpc/id.rs

```diff
@@ -29,3 +29,27 @@ impl Id {
         Id::Str(uuid.to_string())
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::test::test_serialization_roundtrip;
+
+    #[test]
+    fn round_tripping_id() {
```

Comment:
Can we name this so it's more on point?

```suggestion
    fn json_id() {
```

Reply:
I can change it but note that this test will actually shows up as:
`test rpc::id::tests::round_tripping_id ... ok`

which seems clear that we are roundtrippin the rpc id (and all the rpc uses JSON rpc). Maybe that whole module should be renamed to `jsonrpc` though.

Reply:
Is `round_tripping_jsonrpc_id` ok for you?

Reply:
Yeah, I was overeager and didn't check the bigger picture. It's cool.